POEApi.Transport/TaskThrottle.cs

```diff
@@ -55,7 +61,7 @@ public void StartTask()
                     }
 
                     RemvoeExpiredTasks();
```

Comment:
Appreciate it's not your typo, could you adjust please.

Reply:
Done. (I rewrote the class, so, ultimately, it is my typo. :) )